es/i18n/intro/login-es.html

```diff
@@ -2,6 +2,10 @@
 <div class="intro-title">
 <center><h3>Iniciar sesión mediante token anónimo (OPcode)</h3></center>
 
+<div class="save-button">
+  <center><div class="col" ng-click="saveLogin('{{randomToken}}')" class="control-icon-button"> <br>Guarde su OPcode <i class="ion-ios-download-outline" style="font-size: 200%"></i></div></center>
+</div>
+
```

Comment:
This should be below line 10.
Note that this file and the file that you are trying to check in to `e-mission-phone` are subtly different
https://github.com/e-mission/e-mission-phone/pull/863/files#diff-0e3ef9886230bc415ac090d1f91f42cd04679f2059358b0ac87e453218b24f39R6
and both are wrong 😦 

This has the save button above the OPcode, and the other one does not have the `program_or_study=='study' fix.
This is why it is important to not check in two copies of the file, and to ensure the version that you are committing is correct.